styles.css

```
@@ -0,0 +1,108 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    align-items: center;
+    font-family: 'Roboto', sans-serif;
+}
+
+h2 {
+    width: 100%;
+    text-align: center;
+}
+nav {
+    height: 50px;
+    background-color:rgb(230, 249, 253);
+    width: 100%;
+    text-align: center;
+    display: flex;
+
+}
+a {
+    text-decoration: none;
+    padding: 20px;
+    font-weight: bold;
+    color:rgb(50, 48, 48)
+}
+
+header {
+    background-color: ghostwhite;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    
+}
```

```suggestion
}

```
Looks like the prettier extension for your visual studio code is not working on saving the files.